web/src/components/core/Description.jsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) [2023] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+import { Popover } from "@patternfly/react-core";
+
+/**
+ * Displays details popup after clicking the children elements
+ * @component
+ *
+ * @param {object} description content displayed in a popup
+ * @param {object} children the wrapped content
+ */
+export default function Description ({ description, children, ...otherProps }) {
```

Comment:
I'm a bit lost here. I think this component is redundant with the `Attribute` component, which returns a plain label when no description is given. And it's the only place where `Description` is used.

Reply:
The reason is that I'd like to share the code which uses `Popover` for displaying details or help so it can be used elsewhere. When we later decide to use `Tooltip` instead of `Popover` (or change the look) then we need to update just one place.

Reply:
Umm, but we can do the same having only the `Attribute` component, right?. I mean, we can modify the internal implementation of `Attribute` to use a tooltip or to provide other look. And, I would use `Attribute` (or `Tip`) only for the cases in which we want a label with more info. Otherwise, I would use the `Label` directly.

Reply:
The point is that you can use a tooltip also for something else, not only `Label`.

Reply:
Ah, now I got it. Thanks!